execution/engine/eks_engine.go

```diff
@@ -82,16 +83,31 @@ func (ee *EKSExecutionEngine) Execute(td state.Definition, run state.Run) (state
 	job, err := ee.adapter.AdaptFlotillaDefinitionAndRunToJob(td, run, ee.jobSA)
 	result, err := ee.kClient.BatchV1().Jobs(ee.jobNamespace).Create(&job)
 	if err != nil {
+		_ = metrics.Increment(metrics.EngineEKSExecute, []string{
+			string(metrics.StatusFailure),
+			fmt.Sprintf("run_id:%s", run.RunID),
+			fmt.Sprintf("definition_id:%s", run.DefinitionID),
```

Comment:
remove the run_id and definition_id - it will explode the cardinality space. And we can always query this information from the DB on failed jobs.